WTWSMS/common/event_modifiers/WtWSMS_event_modifiers.txt

```diff
@@ -210,27 +210,8 @@ persecuting_nestorians = { icon = 15 }
 
 enforced_local_script = { icon = 15 }
 
-latin_official_language = {
-	global_revolt_risk = 0.1
-	city_vassal_tax_modifier = -0.05
-	temple_vassal_tax_modifier = 0.01
-	culture_flex = -0.05 # Tolerance to other cultures.
-}
 greek_official_language = {
-	global_revolt_risk = -0.05
-	city_vassal_tax_modifier = 0.01
-	temple_vassal_tax_modifier = -0.05
-}
-latin_official_language_2 = {
-	global_revolt_risk = 0.15
-	city_vassal_tax_modifier = -0.1
-	temple_vassal_tax_modifier = 0.05
-	culture_flex = -0.2
-}
-greek_official_language_2 = {
-	global_revolt_risk = -0.05
-	city_vassal_tax_modifier = 0.05
-	temple_vassal_tax_modifier = -0.1
+	city_tax_modifier = 0.05
```

Comment:
What do you think about adding some malus to counterbalance?
e.g. monthly_character_prestige = -0.05

Reply:
The malus is the one time hit at the moment.

Reply:
I like that it requires the themata system, but does it exist? I have searched the mod files and I can't find other occurences of `themata_system` flag. Is it planned?

Reply:
Planned.